<ui version="4.0" >
 <class>matGuide</class>
 <widget class="QDialog" name="matGuide" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Maturity Guide</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>220</y>
     <width>191</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QPushButton" name="mat1Btn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>PushButton</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <property name="autoExclusive" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mat2Btn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>PushButton</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <property name="autoExclusive" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mat3Btn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>110</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>PushButton</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <property name="autoExclusive" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mat4Btn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>160</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>PushButton</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <property name="autoExclusive" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mat8Btn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>360</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>PushButton</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <property name="autoExclusive" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mat7Btn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>310</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>PushButton</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <property name="autoExclusive" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mat6Btn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>260</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>PushButton</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <property name="autoExclusive" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mat5Btn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>210</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>PushButton</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <property name="autoExclusive" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>100</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QPushButton" name="femaleBtn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Female</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <property name="autoExclusive" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="maleBtn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Male</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
    <property name="autoExclusive" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="picLabel" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>10</y>
     <width>691</width>
     <height>461</height>
    </rect>
   </property>
   <property name="autoFillBackground" >
    <bool>true</bool>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="descLabel" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>470</y>
     <width>691</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground" >
    <bool>true</bool>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="wordWrap" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>560</y>
     <width>691</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QPushButton" name="exitBtn" >
    <property name="geometry" >
     <rect>
      <x>530</x>
      <y>10</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Done</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBtn" >
    <property name="geometry" >
     <rect>
      <x>270</x>
      <y>10</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Next Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevBtn" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>Previous Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="spcLabel" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="matTabLabel" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>50</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="acceptDrops" >
    <bool>true</bool>
   </property>
   <property name="autoFillBackground" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>370</y>
     <width>69</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Maturity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>400</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Stage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>140</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Fish</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Sex</string>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>60</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Maturity Table </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
